src/pages/create-course/CreateCourse.tsx

```diff
@@ -42,7 +42,7 @@ const CreateCourse = () => {
 
   const createNewSection = () => {
     const newSectionData = { ...sectionInitialData }
-    newSectionData.id = data.sections.length
+    newSectionData.id = Date.now() - data.sections.length
```

Comment:
You can use one of them. I suppose `Date.now()` or just use `data.sections.length`. I just don't understand this subtraction logic for id. we can leave without data.sections.length or use Date.now() for id ?